spec/serializers/disability_claim_detail_serializer_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe DisabilityClaimDetailSerializer, type: :serializer do
+  let(:disability_claim) { create(:disability_claim) }
+  let(:data) { JSON.parse(subject)['data'] }
+  let(:attributes) { data['attributes'] }
+  subject { serialize(disability_claim, serializer_class: DisabilityClaimDetailSerializer) }
+
+  it 'should include id' do
+    expect(data['id']).to eq(disability_claim.id.to_s)
+  end
+
+  KEYS = %w( evss_id date_filed min_est_date max_est_date
+             phase_change_date open waiver_submitted contention_list
+             va_representative events_timeline development_letter_sent
+             decision_letter_sent).freeze
+
+  it "shouldn't include any extra attributes" do
+    expect(attributes.keys.sort).to eq(KEYS.sort)
+  end
```

Comment:
These tests are covering the same thing that the schema matching covers as we are using strict validation. Maybe add a test that the tracked items are in reverse chron order and contain a certain number of items? Also that date fields are in the format you're expecting (they were `%m/%d/%y` but looks like they are now `%Y-%m-%d` after being converted to Date objects, that may be testable in the schema as well but not sure what the options are for testing date formats)